test/unit/dialects/postgres/query-generator.test.js

```diff
@@ -1331,5 +1331,53 @@ if (dialect.startsWith('postgres')) {
         });
       });
     });
+
+    describe('Custom Schema Functions', () => {
+      beforeEach(function () {
+        this.queryGenerator = new QueryGenerator({
+          sequelize: this.sequelize,
+          _dialect: this.sequelize.dialect,
+        });
+        // Add custom schema to sequelize object
+        this.queryGenerator.sequelize.options.schema = 'custom';
+      });
+      afterEach(function () {
+        // Remove custom schema in order to not interfere with other tests
+        this.queryGenerator.sequelize.options.schema = null;
+      });
+
+      const customSchemaSuites = {
+        showTablesQuery: [
+          {
+            title: 'showTablesQuery should use correct custom schema',
+            arguments: [],
+            expectation: 'SELECT table_name FROM information_schema.tables WHERE table_schema = \'custom\' AND table_type LIKE \'%TABLE\' AND table_name != \'spatial_ref_sys\';',
```

Comment:
Use a template string here to remove the need to escape `'`

Reply:
fixed